src/main/scala/nl.knaw.dans.easy.dd2d/DansDeposit2ToDataverseApp.scala

```diff
@@ -31,15 +33,18 @@ class DansDeposit2ToDataverseApp(configuration: Configuration) extends DebugEnha
     serviceUri = configuration.validatorServiceUrl,
     connTimeoutMs = configuration.validatorConnectionTimeoutMs,
     readTimeoutMs = configuration.validatorReadTimeoutMs)
-  private val inboxWatcher =
+  private val inboxWatcher = {
+    initOutboxDirs(configuration.outboxDir)
```

Comment:
@ChrisMuller80. You wouldn't want this here. What if the outbox dir is not empty? Please, think through all the scenarios. This would probably be an application manager who restarts the service. It should definitely *not* fail to restart because the outbox is already filled with previously processed SWORD-deposits.